apps/backend/src/users/users.controller.ts

```diff
@@ -10,4 +10,12 @@ export class UsersController {
   getAllUsers() {
     return this.usersService.findAll();
   }
+
+  @Put(':userId')
+  async updateUser(
+    @Body() updateUserDTO: UpdateUserDTO,
+    @Param('userId') userId: string,
```

Comment:
:pick: / :thought_balloon: I think it might be better if `userId`s are `number`s because they should just be auto-incrementing numbers right?